game/scripts/vscripts/items/charge_bkb.lua

```diff
@@ -97,3 +97,7 @@ function modifier_charge_bkb_on_created(keys)
 --      return 10.0
 --   end)
 end
+
+item_charge_bkb_2 = item_charge_bkb
```

Comment:
There is no item_charge_bkb.

This item is data driven and i don't really know how it works